src/Microsoft.VisualStudio.Services.Agent/ThrottlingReportHandler.cs

```diff
@@ -45,13 +45,16 @@ protected async override Task<HttpResponseMessage> SendAsync(HttpRequestMessage
             // Inspect whether response has throttling information
             IEnumerable<string> vssRequestDelayed = null;
             IEnumerable<string> vssRequestQuotaReset = null;
+
             if (response.Headers.TryGetValues(HttpHeaders.VssRateLimitDelay, out vssRequestDelayed) &&
                 response.Headers.TryGetValues(HttpHeaders.VssRateLimitReset, out vssRequestQuotaReset) &&
                 !string.IsNullOrEmpty(vssRequestDelayed.FirstOrDefault()) &&
                 !string.IsNullOrEmpty(vssRequestQuotaReset.FirstOrDefault()))
             {
-                TimeSpan delay = TimeSpan.FromMilliseconds(int.Parse(vssRequestDelayed.First()));
-                DateTime expiration = DateTime.Parse(vssRequestQuotaReset.First());
+                TimeSpan delay = TimeSpan.FromMilliseconds(double.Parse(vssRequestDelayed.First()));
```

Comment:
It appears that VssRateLimitDelay used to be an int, but now it's a decimal value. Fixed the parsing so it doesn't throw.

Reply:
sweet....